### ISSUE TITLE
[INFO] "Error pinging docker registry registry-1.docker.io" on Raspberry Pi 3

### ISSUE THREAD

**Issue by mrclschstr** [crazy-max/diun#356]
First of all: Thanks for your work on diun!

I have more of an info than an issue here for other fellow sufferers. I'm running the diun docker container on a Raspberry Pi 3 (linux/arm/v7). I was experiencing the same issues mentioned in #285: `Cannot get image digest from HEAD request: error pinging docker registry registry-1.docker.io`. I could not ping other domains as well.

I finally found the solution here: https://github.com/alpinelinux/docker-alpine/issues/135#issuecomment-812287338

TL;DR: It's a bug since alpine 3.13 (alpine 3.12 works fine...). Just execute the following commands on your RPi to fix the bug for Raspbian:

1. Download: wget `https://ftp.debian.org/debian/pool/main/libs/libseccomp/libseccomp2_2.5.1-1_armhf.deb`
2. Install: `sudo dpkg -i libseccomp2_2.5.1-1_armhf.deb`

Now diun should run perfectly fine 😃

**Comment by crazy-max** [comment]
Yes indeed you need at least libseccomp2 2.4.2 and Docker 19.03.9.
